src/sentry/static/sentry/app/components/events/interfaces/threads.jsx

```diff
@@ -113,17 +114,12 @@ function getIntendedStackView(thread, event) {
 }
 
 function findBestThread(threads) {
-  for (const thread of threads) {
-    if (thread.crashed) {
-      return thread;
-    }
-  }
-  for (const thread of threads) {
-    if (thread.stacktrace) {
-      return thread;
-    }
-  }
-  return threads[0];
+  return (
+    threads.find(thread => thread.crashed) ||
```

Comment:
We maybe want a comment here that there is a priority involved with finding the "best" thread (I could see someone in the future trying to simplify this as finding any crashed|errored|stacktrace thread.

Reply:
Good idea, done.